runtime/doc/change.txt

```diff
@@ -427,6 +427,12 @@ Note similarly, when 'nrformats' includes "bin", binary numbers with a leading
 '0x' or '0X' can be interpreted as hexadecimal rather than binary since '0b'
 are valid hexadecimal digits.
 
+When the number under the cursor is too big to fit into 32 or 64 bit
+(depending on how Vim was build), it will be rounded off to the nearest number
+that can be represented, and the addition/subtraction is skipped.  E.g. with
+64 bit support using CTRL-X on 18446744073709551616 results in
+18446744073709551615.  Same for larger numbers, such as 18446744073709551618.
+
```

Comment:
I think this is always 64-bits in Nvim? `uvarnumber_T`?

Reply:
```suggestion
When the number under the cursor is too big to fit into 64 bit, it will be rounded off to the nearest number
that can be represented, and the addition/subtraction is skipped.  E.g. using CTRL-X on 18446744073709551616 results in
18446744073709551615.  Same for larger numbers, such as 18446744073709551618.

```
Like so?

Reply:
Looks good to me! :+1: 
Maybe `bit` -> `bits` if I'm allowed to be pedantic :innocent: